Doc/whatsnew/3.12.rst

```diff
@@ -102,7 +102,11 @@ Deprecated
 Removed
 =======
 
-
+* Removed randrange() functionality deprecated since Python 3.10.  Formerly,
```

Comment:
I think it would be better to move to section "Changes in the Python API".

Reply:
Okay.